recipe/conda_build_config.yaml

```diff
@@ -46,6 +70,11 @@ cudnn:
   - 8      # [linux64 or win]           # CUDA 11.1
   - 8      # [linux64 or win]           # CUDA 11.2
 
+  - undefined   # [linux and ppc64le]        # CUDA 10.2
+  - undefined   # [linux and ppc64le]        # CUDA 11.0
+  - undefined   # [linux and ppc64le]        # CUDA 11.1
+  - undefined   # [linux and ppc64le]        # CUDA 11.2
```

Comment:
We should probably package these in the future

Reply:
Created an issue there so we don't forget!